packages/core/src/delegation/DelegationNode.spec.ts

```diff
@@ -27,7 +27,7 @@ let nodes: Record<string, DelegationNode> = {}
 jest.mock('./DelegationNode.chain', () => {
   return {
     getChildren: jest.fn(async (node: DelegationNode) =>
-      node.childrenIds.map((id) => nodes[id] || null)
+      node.childrenIds.map((id: IDelegationNode['id']) => nodes[id] || null)
```

Comment:
Is this a new eslint rule, that we always have to provide the type for these array function parameters, besides typescript being able to easily infer them?

Reply:
Removed.